### ISSUE TITLE
9.2.5 Issue

### ISSUE THREAD

**Issue by zaphon** [kemayo/wow-handynotes-shadowlandstreasures#13]
```
527x Usage: local sources = C_TransmogCollection.GetAppearanceSources(appearanceID, categoryType, transmogLocation)
[string "=[C]"]: in function `GetAppearanceSources'
[string "@HandyNotes_ShadowlandsTreasures\handler\config.lua"]:404: in function <...s\HandyNotes_ShadowlandsTreasures\handler\config.lua:383>
[string "=(tail call)"]: ?
[string "@HandyNotes_ShadowlandsTreasures\handler\config.lua"]:318: in function <...s\HandyNotes_ShadowlandsTreasures\handler\config.lua:316>
[string "=(tail call)"]: ?
[string "@HandyNotes_ShadowlandsTreasures\handler\config.lua"]:496: in function <...s\HandyNotes_ShadowlandsTreasures\handler\config.lua:494>
[string "@HandyNotes_ShadowlandsTreasures\handler\config.lua"]:622: in function `should_show_point'
[string "@HandyNotes_ShadowlandsTreasures\handler\handler.lua"]:990: in function `(for generator)'
[string "@HandyNotes\HandyNotes-v1.6.5.lua"]:435: in function `UpdateMinimapPlugin'
[string "@HandyNotes\HandyNotes-v1.6.5.lua"]:494: in function `?'
[string "@Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:119: in function <...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:119>
[string "=[C]"]: ?
[string "@Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:29: in function <...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:25>
[string "@Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:64: in function `SendMessage'
[string "@HandyNotes_ShadowlandsTreasures\handler\handler.lua"]:1053: in function `Refresh'
[string "@HandyNotes_ShadowlandsTreasures\handler\handler.lua"]:1049: in function <...\HandyNotes_ShadowlandsTreasures\handler\handler.lua:1044>
```

**Comment by zaphon** [comment]
Actually appears all you handynotes plugins make the same call.

**Comment by kemayo** [comment]
Yeah, I'm looking into it at the moment. Blizzard changed the transmog APIs to require some extra parameters.

**Comment by kemayo** [comment]
Fixed as of 46996d9e1d67a1e30701186feee699a91dea760d. I'll need to get the fix out in everything else as well now.
